docs/docs/basic-usage.md

```diff
@@ -134,6 +134,15 @@ the dependencies installed are still working even if your dependencies released
 
     For libraries it is not necessary to commit the lock file.
 
+### Installing dependencies only
+
+Poetry will install all of a project's dependencies and the current project by default.
```

Comment:
Hey,

I guess it would be a good idea to say, that by default the current project is installed in development/editable mode. What do you think?

fin swimmer

Reply:
Great idea, I've worked in your suggestion. Thanks!